content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -465,18 +809,49 @@ model Post {
 }
 ```
 
-`@default` attributes typically represent `DEFAULT` values in the underlying database. However, functions like `cuid()` and `uuid()` are provided by Prisma's [query engine](../prisma-engines/query-engine/) and are therefore not "visible" in the underlying database schema.
+</tab>
+<tab>
+
+```prisma
+model Post {
+  id         String         @id @default(dbgenerated()) @db.ObjectId @map("_id")
+|  createdAt  DateTime    @default(now())
+  title      String
+|  published  Boolean     @default(false)
+  author     User        @relation(fields: [authorId], references: [id])
+  authorId   String      @db.ObjectId
+  categories Category[]  @relation(references: [id])
+}
+```
+
+</tab>
+</TabbedContent>
+
+
+`@default` attributes either:
+
+* Represent `DEFAULT` values in the underlying database (relational databases only) _or_
+* Use a Prisma-level function. For example, `cuid()` and `uuid()` are provided by Prisma's [query engine](../prisma-engines/query-engine/) for all connectors.
 
 Default values can be:
 
 - Static values that correspond to the field type, such as `5` (`Int`), `Hello` (`String`), or `false` (`Boolean`)
 - [Functions](#using-functions), such as [`now()`](../../../reference/api-reference/prisma-schema-reference#now) or [`uuid()`](../../../reference/api-reference/prisma-schema-reference#uuid)
 
+<Admonition type="info">
+
+Refer to the [attribute function reference documentation](../../../reference/api-reference/prisma-schema-reference#attribute-functions) <span class="api"></span> for information about connector support for functions.
+
+</Admonition>
+
 ### Defining a unique field
 
 You can add unique attributes to your models to be able to uniquely identify individual records of that model. Unique attributes can be defined on a single field using [`@unique`](../../../reference/api-reference/prisma-schema-reference#unique) <span class="api"></span> attribute, or on multiple fields (also called composite or compound unique constraints) using the [`@@unique`](../../../reference/api-reference/prisma-schema-reference#unique-1) <span class="api"></span> attribute.
 
-In the following example, the value of the `email` field must be unique:
+In the following example, the value of the `email` field must be unique. Because the `email` field functions as a unique identifier, it must be mandatory, and (MongoDB only) decorated with the `@map("_id")` attribute:
```

Comment:
```suggestion
In the following example, the value of the `email` field must be unique. Because the `email` field functions as a unique identifier for the model (which is required by Prisma), it must be mandatory, and (MongoDB only) decorated with the `@map("_id")` attribute:
```

Reply:
Changed this section but have incorporated the 'for the model (which is required by Prisma)' part